### ISSUE TITLE
Minor typo in wfpiconsole.sh

### ISSUE THREAD

**Issue by ctgreybeard** [peted-davis/WeatherFlow_PiConsole#22]
```
"\\n\\nThanks for checking out the WeatherFlow PiConsole. This script will guide you through the installation process on your Raspbery Pi." ${r} ${c}
```

Raspberry is misspelled.

**Comment by peted-davis** [comment]
Thanks for the spot! I'll get this corrected (sorry for the slow response as well - seems like notifications for new issues stopped working...)

**Comment by peted-davis** [comment]
Fixed in v3.8
